src/solver/conv_cellfft.cpp

```diff
@@ -0,0 +1,603 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2020 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+#include <sstream>
+#include <miopen/config.h>
+#include <miopen/kernel_info.hpp>
+#include <miopen/gcn_asm_utils.hpp>
+#include <miopen/solver.hpp>
+#include <miopen/env.hpp>
+#include <miopen/kernel_build_params.hpp>
+#include <miopen/conv/tensors.hpp>
+#include <boost/any.hpp>
+#include <miopen/conv/invokers/cellfft.hpp>
+
+// clang-format off
+#define START_R2C_S   0
```

Comment:
```suggestion
constexpr size_t start_r2c_s = 0;
```

That would provide a nicely typed value. The same applies to the rest of the defines.

Reply:
Ok